baseplate/sidecars/live_data_watcher.py

```diff
@@ -135,8 +137,15 @@ def _load_from_s3(data: bytes) -> bytes:
         )
         raise LoaderException from e
 
+    # Client needs to be anonymous/unsigned or boto3 will try to read the local credentials
+    # on the service pods. And - due to an AWS quirk - any request that comes in signed with
+    # credentails will profile for permissions for the resource being requested EVEN if the
+    # resource is public. In other words, this means that a given service could not access
+    # public resources belonging to another cluster/AWS account unless the request credentials
+    # were unsigned.
     s3_client = boto3.client(
         "s3",
+        config=Config(signature_version=UNSIGNED),
```

Comment:
Note: this makes every s3 client use unsigned requests. Right now the only customer is Experiments, but in the future it could be other services wanting to make use of this pathway.

Do we want to have an explicit check here to see if the use-case is experiments? And if and only if so THEN we make it unsigned?

The con here is that we'd be strict checking some field/name to be able to tell if it's Experiments which goes against the goal of making the sidecar logic more broad...and branching logic is messy in general BUT if there are security concerns at play here, we should definitely do it.

Reply:
Another alternative I just thought of would be to include another special key-value in the ZK file (e.g. `"anon":true`)) which would let the live-data-fetcher know to instantiate the s3_client this way.